http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -708,6 +717,16 @@ public void setEnabled(boolean enabled) {
             this.enabled = enabled;
         }
 
+        /**
+         * Sets whether localhost pass-through is enabled. Default value {@value #DEFAULT_LOCALHOST_PASS_THROUGH}.
+         * Setting this to true will allow requests to be made to localhost from any origin.
+         *
+         * @param localhostPassThrough True if localhost pass-through is enabled
+         */
+        public void setLocalhostPassThrough(boolean localhostPassThrough) {
```

Comment:
add `@since` declarations to the new methods

Reply:
👍